### PR TITLE
Add a news post for playtest-20160904.

### PR DESCRIPTION
**Pull request by pchote**

### CODE REVIEW

File: content/news/playtest-20160904.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+Today we're happy to finally announce the first release candidate for our next OpenRA release.
+The new playtest contains over 1200 commits by 38 authors (including 11 first-time committers, thanks to all of you!).
+
+We rely on players to thoroughly test the playtest builds to catch any bugs or balance regressions before the final release, so head over to the [download page](/download/) to try it out today!. If you encounter a bug, or have any feedback, then please create a ticket on our [Github issue tracker](http://bugs.open-ra.org), [find us on IRC](http://webchat.freenode.net/?channels=openra), create a topic [on our community forum](www.sleipnirstuff.com/forum/viewforum.php?f=80), or leave a message in the comments below.
```
Comment (Phrohdoh) [inline content/news/playtest-20160904.md]: `!.` should be `!`

**Comment by pchote** [comment]
Added gamenews.  Just need the release ID now.

File: content/news/playtest-20160904.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+Today we're happy to finally announce the first release candidate for our next OpenRA release.
+The new playtest contains over 1200 commits by 38 authors (including 11 first-time committers, thanks to all of you!).
+
+We rely on players to thoroughly test the playtest builds to catch any bugs or balance regressions before the final release, so head over to the [download page](/download/) to try it out today! If you encounter a bug, or have any feedback, then please create a ticket on our [Github issue tracker](http://bugs.open-ra.org), [find us on IRC](http://webchat.freenode.net/?channels=openra), create a topic [on our community forum](www.sleipnirstuff.com/forum/viewforum.php?f=80), or leave a message in the comments below.
```
Comment (obrakmann) [inline content/news/playtest-20160904.md]: http://bugs.open-ra.org? -> http://bugs.openra.net

File: content/news/playtest-20160904.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+A lot of work in this release has focused on improving our core engine technology and paying down technical debt. The first place you will notice this is the mod chooser, which now understands the various install disks and digitial distributions of the original games. It can now automatically find and install all of the assets used by the mods, including the full motion videos and the additional music tracks included with the expansion disks.
+
+<div style="text-align:center" markdown="1">
+![New content installer](/images/news/20160904-cnc-contentinstall.png)
+</div>
+
+Because we have changed which assets are copied, all players will need to reinstall (or redownload) the assets for this playtest. These will be used for the future playtests and releases, so only needs to be done once.
+
+The new FMV installation support has arrived just in time (some would argue far too late!), because this playtest includes 7 more campaign missions for TD and one new campaign mission each for RA and D2K. RA also includes three custom multiplayer missions that can be selected from the mission chooser (for single-player) or the multiplayer lobby (for cooperative play).
```
Comment (obrakmann) [inline content/news/playtest-20160904.md]: I'm never consistent with this myself, but at least in one sentence you should decide on whether to have a hyphen in (single|multi)-player or not ;)

File: content/news/playtest-20160904.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+<div style="text-align:center" markdown="1">
+![Burning trees](/images/news/20160904-cnc-burningtrees.png)
+
+We have restored the ability for flame weapons to burn trees down to charred husks.
+<br />
+Try the playtest today to find all the other cosmetic improvements!
+</div>
+
+Check out [the full changelog](http://changelog.openra.net) for more details, and head over to the [download page](/download/) to try it for yourself!
```
Comment (obrakmann) [inline content/news/playtest-20160904.md]: I think you can leave out the download link here, you already mentioned it in the second paragraph.

**Comment by obrakmann** [comment]
Looks good otherwise, :+1:

File: content/news/playtest-20160904.md (hunk 4 above)
Comment (pchote) [inline content/news/playtest-20160904.md]: The line feels incomplete with just the changelog part, so i'd prefer to keep this.

Comment (obrakmann) [inline content/news/playtest-20160904.md]: Here's the changelog permalink: https://github.com/OpenRA/OpenRA/wiki/Changelog/c25c6528e4eeca1e9e36c2b25c2fbae8b4d9c363

**Comment by pchote** [comment]
Updated with changelog link and release ID.  :shipit:

**Pull request merged**